tests/test_openjdk.py

```diff
@@ -156,3 +174,65 @@ def test_jdk_extended(
 
     for check in params.expected_err_strings:
         assert check in testout.stderr
+
+
+@pytest.mark.parametrize(
+    "container_per_test",
+    CONTAINER_IMAGES_CASSANDRA,
+    indirect=["container_per_test"],
+)
+@pytest.mark.parametrize(
+    "cassandra_version", [(os.getenv("APPLICATION_VERSION", "4.0.6"))]
+)
+def test_jdk_cassandra(
+    container_per_test,
+    cassandra_version,
+    host,
+    container_runtime: OciRuntimeBase,
+):
+    """Starts the Cassandra DB and executes some write and read tests
+    using the cassandra-stress
+    """
+
+    container_per_test.connection.run_expect(
+        [0],
+        f"curl -OL https://downloads.apache.org/cassandra/{cassandra_version}/apache-cassandra-{cassandra_version}-bin.tar.gz",
+    )
+
+    container_per_test.connection.run_expect(
+        [0], f"tar xzvf apache-cassandra-{cassandra_version}-bin.tar.gz"
+    )
+
+    container_per_test.connection.run_expect(
+        [0],
+        f"export JAVA_HOME=/usr/ && cd apache-cassandra-{cassandra_version}/ && bin/cassandra -R >/var/log/cassandra.log",
+    )
+
+    found = False
+    for t in range(20):
+        time.sleep(10)
+        if (
+            "state jump to NORMAL"
+            in container_per_test.connection.check_output(
+                "cat /var/log/cassandra.log"
+            )
+        ):
+            found = True
+            break
+
+    assert (
+        "state jump to NORMAL"
+        in container_per_test.connection.check_output(
+            "cat /var/log/cassandra.log"
+        )
+    )
```

Comment:
Why this duplication and not just `assert found`?

Reply:
Done